docs/characterization/xeb_coherent_noise.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>This notebook will eventually end up on the site. Please annotate the types with packages, e.g <code>cirq.PhasedFSimGate</code>  - this will create automated links to the names on the reference docs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3706/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
cool, done